examples/evm/call-contract-with-token/index.js

```diff
@@ -30,9 +26,7 @@ async function deploy(chain, wallet) {
 
 async function execute(chains, wallet, options) {
     const args = options.args || [];
-    const getGasPrice = options.getGasPrice;
-    const source = chains.find((chain) => chain.name === (args[0] || 'Avalanche'));
-    const destination = chains.find((chain) => chain.name === (args[1] || 'Fantom'));
+    const { source, destination, calculateBridgeFee } = options;
```

Comment:
It comes from [here](https://github.com/axelarnetwork/axelar-examples/blob/e50cba0f4492dd352aae51a8b0c574124bf0b631/scripts/libs/execute.js#L44-L51). That means it will be available as long as the example is run by `npm run execute <example>` command.